### PR TITLE
agent-ui: refactor test data builders and add new ones

### PR DESCRIPTION
**Pull request by t-bast**
This will be helpful to create complex test state in our components

### CODE REVIEW

File: packages/agent-ui/src/test/builders/state/index.js [hunk 1]
```diff
@@ -0,0 +1,3 @@
+export { default as TestProcessBuilder } from './testProcessBuilder';
+export { default as TestAgentBuilder } from './testAgentBuilder';
+export { default as testStateBuilder } from './testStateBuilder';
```
Comment (jeremie-stratumn) [inline packages/agent-ui/src/test/builders/state/index.js]: capitalize first letter pls: `TestStateBuilder`

File: packages/agent-ui/src/test/builders/state/testStateBuilder.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+/**
+ * This data builder class allows tests to build state objects that look like
+ * what the redux state will contain.
+ */
+export default class {
```
Comment (jeremie-stratumn) [inline packages/agent-ui/src/test/builders/state/testStateBuilder.js]: do you want to add more functions to this class?

File: packages/agent-ui/src/test/builders/state/index.js (hunk 1 above)
Comment (t-bast) [reply packages/agent-ui/src/test/builders/state/index.js]: thanks for catching it, definitely my VS code linting it too fast :)

File: packages/agent-ui/src/test/builders/state/testStateBuilder.js (hunk 2 above)
Comment (t-bast) [reply packages/agent-ui/src/test/builders/state/testStateBuilder.js]: Indeed, went too fast :)

**Pull request merged**